src/components/sbb-slider/sbb-slider.tsx

```diff
@@ -117,12 +121,13 @@ export class SbbSlider implements ComponentInterface {
     const mathFraction: number = (value - min) / (max - min);
     this._valueFraction =
       isNaN(mathFraction) || mathFraction < 0 ? 0 : mathFraction > 1 ? 1 : mathFraction;
-    this._syncValues();
+    this._syncValues(value);
   }
 
-  @Listen('keydown')
-  public async handleKeydown(event: KeyboardEvent): Promise<void> {
-    if (this.disabled || this.readonly || (await isEventPrevented(event))) {
+  private async _handleKeydown(event: KeyboardEvent): Promise<void> {
+    // FIXME the third condition in the if doesn't prevent from scrolling
```

Comment:
Can't it be fixed? "Third condition" would not be a good hint for what to do later

Reply:
This was meant to be discussed but i forgot to add a comment this morning :'(

Apparently, the `(await isEventPrevented(event))` is not working anymore (maybe something related to the removal of the Listen annotation?), and fo fixing it I added a `event.preventDefault` before the if condition; if this sounds ok to you, the fixme can be simply removed.

Reply:
From what I remember, with the native slider/range input, if the keydown event is prevented, the resulting change is prevented.

Reply:
As it is now, when i change values with arrows I can see both the input/change events in the Storybook 'Actions' tab